person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../decorators/nameable'
+
 class Person
```

Comment:
- [ ] As mentioned in the previous review, please make sure the person class inherits from the nameable class and has a method called correct name.